### PR TITLE
fix: add validation to prevent duplicated local ports

### PR DESCRIPTION
**Pull request by dnlopes**
This PR addresses an issue when a duplicated port on the Remote configuration was provided.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -140,7 +141,13 @@ func client(args []string) {
 		log.Fatalf("A server and least one remote is required")
 	}
 	config.Server = args[0]
-	config.Remotes = args[1:]
+
+	if err := validateRemotes(args[1:]); err != nil {
+		log.Fatal(err)
+	} else {
```
Comment (bmartins-unit) [inline main.go:147]: why the else?

File: main.go [hunk 2]
```diff
@@ -167,3 +174,35 @@ func client(args []string) {
 		log.Fatal(err)
 	}
 }
+
+// validate the provided Remotes configuration is valid
+func validateRemotes(remotes []string) error {
+	uniqueRemotes := []string{}
+
+	for _, newRemote := range remotes {
```
Comment (bmartins-unit) [inline main.go:182]: can't we create a list/slice with ports and use the "unique" package and check the size ?

**Review by bmartins-unit — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (dnlopes) [reply main.go]: Fixed.

File: main.go (hunk 2 above)
Comment (dnlopes) [reply main.go:182]: For future proof scenarios, in case we want to extend the validations, having it behind a method is easier.

**Review by bmartins-unit — APPROVED** [review]

**Pull request merged**